app/model/CoAFEAuthority.scala

```diff
@@ -16,10 +16,15 @@
 
 package model
 
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
 import play.api.libs.json._
 
-case class CoAFEAuthority(twoFactorAuthOtpId: Option[String])
+case class CoAFEAuthority(twoFactorAuthOtpId: Option[String], enrolmentsUri: String = "", userDetailsLink: String = "")
 
 object CoAFEAuthority {
-  implicit val reads = Json.reads[CoAFEAuthority]
+  implicit val reads =
+    ((JsPath \ "twoFactorAuthOtpId").readNullable[String] and
+      (JsPath \ "enrolments").read[String] and
+      (JsPath \ "userDetailsLink").read[String]).apply(CoAFEAuthority.apply _)
```

Comment:
Custom deserialiser is needed here as `enrolments` field is translated into `enrolmentsUri`.